### DISCUSSION TITLE
How to manually display mantine date picker modal

### DISCUSSION THREAD

**Discussion by arielclement** [mantinedev/mantine#5883] [Q&A] [answered]
hey everyone I am currently using `MonthPickerInput` in my project. I am also displaying an icon by passing it to the `rightSection` property of `MonthPickerInput`.

However I notice that when I click on the icon, `MonthPickerInput` does not automatically display the month picker. Is there a way to trigger it manually by clicking on the icon in my case?

<img width="373" alt="Screenshot 2024-03-07 at 12 48 27" src="https://github.com/mantinedev/mantine/assets/97739122/65911fe6-479f-45d8-bcd0-ef170c084154">

**Comment by rtivital** [comment, marked as answer]
`popoverProps={{ opened: true }}`

**Reply by arielclement** [reply]
Thanks a bunch!
